### PR TITLE
decoded_instructions_temp

### PR DESCRIPTION
**Pull request by eric-laurello**
Will run a clone to get the full table copied to test performance 1-1:
create Transient table solana_dev.silver.decoded_instructions_2  clone solana_dev.silver.decoded_instructions

### CODE REVIEW

File: models/silver/parser/silver__decoded_instructions_2.sql [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+{% if is_incremental() %}
+{{ ref('bronze__streamline_decoded_instructions_2') }} A
+{% else %}
+    {{ ref('bronze__streamline_FR_decoded_instructions_2') }} A
+{% endif %}
+JOIN {{ ref('silver__blocks') }}
+b
+ON A.block_id = b.block_id
+JOIN {{ ref('silver__transactions') }} C
```
Comment (tarikceric) [inline models/silver/parser/silver__decoded_instructions_2.sql:65]: would it make sense to filter down the silver__transactions table to a date range as well? Thinking the join might be more efficient if its currently doing a larger scan of transactions

Comment (eric-laurello) [reply models/silver/parser/silver__decoded_instructions_2.sql:65]: Yeah was trying to think through how to do it, but a backfill could in theory pull in any dates right?

Comment (eric-laurello) [reply models/silver/parser/silver__decoded_instructions_2.sql:65]: what I could do is add the block_timestamp::DATE to the join

Comment (tarikceric) [reply models/silver/parser/silver__decoded_instructions_2.sql:65]: yea good point on the backfill, the range would be large either way. I'll make a separate ticket for improving the join

**Review by tarikceric — APPROVED** [review]

File: models/silver/parser/silver__decoded_instructions_2.sql [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ON A.tx_id = C.tx_id
+AND b.block_timestamp :: DATE = b.block_timestamp :: DATE
```
Comment (tarikceric) [inline models/silver/parser/silver__decoded_instructions_2.sql:67]: should the second be `c.block_timestamp`?

Comment (eric-laurello) [reply models/silver/parser/silver__decoded_instructions_2.sql:67]: good catch

**Pull request merged**